test_check_if_anagrams.py

```diff
@@ -1,6 +1,9 @@
 def are_anagrams(str1, str2):
   # Remove the `return None` code below and add your own code.
-  return None
+  if sum(bytearray(str1.upper())) == sum(bytearray(str2.upper())):
```

Comment:
Well I thought that if I could somehow find the ASCII or byte value of each character in the string (byte array()) then add them together (sum), I could then compare the two strings together. Since anagrams have the same letters in them I figured that they would also both have the same values. I searched through the [documentation](https://docs.python.org/2/library/stdtypes.html#string-methods) for how to get the value of a string. I then changed both strings to uppercase so that the values wouldn't be different even if some of the letters were uppercased. Another similar example might be something like:
def are_anagrams(str1, str2):
	if sum(bytearray(str1.upper())) == sum(bytearray(str2.upper())):
		return True
	else:
		return False
		
print are_anagrams("borerts", "roberts")
print are_anagrams("josh salyer", "JoSh SaLyEr")
print are_anagrams("thicken", "Chicken")
print are_anagrams("not an anagram", "I told you it wasn't")

These would all return true except the last one. I'm not sure if these are the examples you were looking for or not.